cosmo_tester/framework/test_hosts.py

```diff
@@ -882,6 +882,8 @@ def enable_nics(self):
 
         self._logger.info('Adding extra NICs...')
 
+        self.run_command('yum install -y ifupdown', use_sudo=True)
```

Comment:
For this and the other yum install, they shouldn't be in the framework, they should be installed on the image that needs to use them.